api/v1beta1/helmchart_types.go

```diff
@@ -92,6 +97,9 @@ const (
 // the typed referenced object at namespace level.
 type LocalHelmChartSourceReference struct {
 	// APIVersion of the referent.
+	// +kubebuilder:validation:MaxLength=63
```

Comment:
```suggestion
	// +kubebuilder:validation:MaxLength=2048
```

Please change this in all objects with APIVersion fields.